lib/Sabberworm/CSS/Parser.php

```diff
@@ -688,40 +762,12 @@ private function streql($sString1, $sString2, $bCaseInsensitive = true) {
 	}
 
 	private function strtolower($sString) {
-		if ($this->oParserSettings->bMultibyteSupport) {
-			return mb_strtolower($sString, $this->sCharset);
+		if ($this->sTextLibrary === 'mb') {
+			return mb_strtolower($sString, 'utf-8');
```

Comment:
Since the `strtolower` method is only used for identifiers, which, after your change, are always ASCII, just calling `strtolower` should suffice.